### PR TITLE
Updated roster design/player card UI and WUFO front page

### PR DESCRIPTION
**Pull request by alansun25**
- Changed from grid to flex box for responsiveness.
- Added player name to front of card.
- Standardized size of player card to 300px by 300px.
- Added roster title.
- Added flip effect with plater info on the back.
- Changed background color of roster to purple gradient.
- Changed WUFO front page pick to Kaizen and Nick and centered the text "WUFO".

### CODE REVIEW

File: src/components/Player.jsx [hunk 1]
```diff
@@ -1,18 +1,98 @@
...
+        <div 
+          className="absolute top-4 left-2 text-white py-1 px-3 text-2xl font-semibold"
+          style={{ textShadow: '3px 2px #4C1D95' }}
+        >
+          Alan Sun
+        </div>
+        <div 
+          className="absolute top-16 left-8 text-white py-1 px-3 text-md font-medium border-l-4 border-purple-400"
+        >
+          <span className="italic">Nickname:</span> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Lasuna
```
Comment (ritot) [inline src/components/Player.jsx:78]: At some point we probably wanna figure out how to align the text automatically when we dynamically populate the back of the cards

**Review by ritot — APPROVED** [review]
This looks awesome 😎 Love the new navbar and the roster cards are sick!

Something we can discuss later is maybe have the cards flip on hover instead of on click since some people might not know to? On a similar note, maybe have the team dropdown menu in the navbar also expand on hover instead of click, although that one has a down arrow so potentially less confusing

**Comment by ritot** [comment]
Also another thought: is there an easy way to flip the image horizontally using ReactCardFlip? That'd be cool

File: src/components/Player.jsx (hunk 1 above)
Comment (alansun25) [reply src/components/Player.jsx:78]: I think with the way their positioned, they should always be aligned, tho you're right, using two divs in a row would definitely be better practice.

**Comment by alansun25** [comment]
> This looks awesome 😎 Love the new navbar and the roster cards are sick!
> 
> Something we can discuss later is maybe have the cards flip on hover instead of on click since some people might not know to? On a similar note, maybe have the team dropdown menu in the navbar also expand on hover instead of click, although that one has a down arrow so potentially less confusing

I agree, hover would be better. Tho on mobile it would probably still need to be on click yeah?

**Comment by alansun25** [comment]
> Also another thought: is there an easy way to flip the image horizontally using ReactCardFlip? That'd be cool

Yah I can look into that.

**Review by ritot — APPROVED** [review]
This looks amazing 🤩 CMU got nothing on us 😤

**Pull request merged**